docs/usage/configuration.md

```diff
@@ -76,6 +76,8 @@ The branches on which releases should happen. By default **semantic-release** wi
 
 **Note**: Once **semantic-release** is configured, any user with the permission to push commits on one of those branches will be able to publish a release. It is recommended to protect those branches, for example with [GitHub protected branches](https://help.github.com/articles/about-protected-branches).
 
+**Note**: The prerelease can't be named with `/` because it doesn't follow the [Semver spec](https://semver.org/). For example `{ "name": "debug/*", "prerelease": true }` won't work. You have to manually change all invalid characters (`/` in this case) to something else (example: `{ "name": "debug/*", "prerelease": '${name.replace(/\//, "-")}' }`)
```

Comment:
As I already mention in https://github.com/semantic-release/semantic-release/pull/1420#issuecomment-576365278 we don't want to mention a workaround specific for the `/` case.
If we have to amend the doc for each possible invalid character a user tries then it's infinite ans we'll never stop changing this doc.

The generated prerelease id has to be valid by the spec and it's up to the user to read the spec. This is why we verify the format in the code and and print an error message linking to the spec.

If users don't read error message and don't click on the spec link there is no much we can do. We can't just list here every imaginable invalid character.